samples/boards/nrf52/mesh/onoff-app/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: Bluetooth Mesh
+tests:
+  - test:
+      build_only: true
+      platform_whitelist: bbc_microbit nrf51_blenano
+        qemu_x86
```

Comment:
This looks a bit odd. The app is under samples/boards/nrf52/ but there's not a single nRF52 board in the list of boards that CI will build it for? I'd expect only your main target board to be in the platform_whitelist.

Reply:
I didn't understand how the .yaml worked. I'll fix it.